aikau/src/main/resources/alfresco/core/Core.js

```diff
@@ -408,11 +408,15 @@ define(["dojo/_base/declare",
             topics = [topics];
          }
 
+         /*jshint -W059 */
+         var callerName = arguments.callee.caller.name;
```

Comment:
This (`arguments.callee`) has been removed from JavaScript. Browsers still support it, but for how long? Is there really no alternative?